src/asm/parser.y

```diff
@@ -980,8 +980,10 @@ load		: T_POP_LOAD sectmod string T_COMMA sectiontype sectorg sectattrs {
 
 rept		: T_POP_REPT uconst T_NEWLINE {
 			lexer_CaptureRept(&captureBody);
-		} T_NEWLINE {
-			fstk_RunRept($2, captureBody.lineNo, captureBody.body, captureBody.size);
+		} endofline {
+			if (!captureBody.unterminated)
```

Comment:
Since Bison [allows midrules to have semantic values](https://www.gnu.org/software/bison/manual/html_node/Typed-Midrule-Actions.html), could the `lexer_Capture*` functions be instead made to return their success status instead?

Reply:
Sure, I've added that (though it can't use the syntax new to Bison 3.1).